libs/labelbox/src/labelbox/client.py

```diff
@@ -590,6 +590,55 @@ def _create(self, db_object_type, data):
         res = res["create%s" % db_object_type.type_name()]
         return db_object_type(self, res)
 
+    def create_model_config(self, name: str, inference_params: dict, model_id: str) -> str:
+        """ Creates a new model config with the given params.
+            Model configs are scoped to organizations, and can be reused between projects.
+
+        Args:
+            name (str): Name of the model config
+            inference_params (dict): JSON of model configuration parameters.
+            model_id (str): ID of model to configure
+
+        Returns:
+            str, id of the created model config
+        """
+
+        query = """mutation CreateModelConfigPyApi($modelId: ID!, $inferenceParams: Json!, $name: String!)  {
+                    createModelConfig(input: {modelId: $modelId, inferenceParams: $inferenceParams, name: $name}) {
+                        modelId
+                        inferenceParams
+                        name
+                    }
+                }"""
+        params = {
+            "modelId": model_id,
+            "inferenceParams": inference_params,
+            "name": name
+        }
+        result = self.execute(query, params)
+        return result['createModelConfig']['modelConfigId']
```

Comment:
you're not seem to be selecting "modelConfigId" in the `createModelConfig` response, so I suppose we get a KeyError here?
Also, I believe it should be simply `id`, and since that's all you want to return here, you can also simplify the selection of response fields to `id` only.

Reply:
I think I need the query key here (`createModelConfig`), at least every other example we have of indexing into a client execute result uses it (I'm having trouble actually calling these queries right now). Agreed about the ID field, that's fixed.